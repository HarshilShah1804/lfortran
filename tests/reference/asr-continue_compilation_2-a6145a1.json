{
    "basename": "asr-continue_compilation_2-a6145a1",
    "cmd": "lfortran --semantics-only --continue-compilation --no-color {infile}",
    "infile": "tests/errors/continue_compilation_2.f90",
    "infile_hash": "7ce55f8b446d9eb6c082a13b828fbdbb8edcbdb8ffe81e77fc44e578",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-continue_compilation_2-a6145a1.stderr",
    "stderr_hash": "ab2434ddc76881d183094599beb3405d0e193d629e84f54dcf1ba68d",
    "returncode": 1
}