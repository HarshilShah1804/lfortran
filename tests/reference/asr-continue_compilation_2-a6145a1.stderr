semantic error: Duplicate DIMENSION attribute specified
  --> tests/errors/continue_compilation_2.f90:22:12
   |
22 |     common /rowns/ rowns(209)
   |            ^^^^^^^^^^^^^^^^^^ 

semantic error: kind argument of `int` intrinsic must be a scalar
   --> tests/errors/continue_compilation_2.f90:282:54
    |
282 |     integer(8), parameter :: ar1(3) = int([1, 2, 3], [8, 8, 8])
    |                                                      ^^^^^^^^^ 

semantic error: kind argument of `int` intrinsic must be a scalar
   --> tests/errors/continue_compilation_2.f90:284:54
    |
284 |     integer(8), parameter :: ar2(3) = int([1, 2, 3], [8, 8, 8])
    |                                                      ^^^^^^^^^ 

semantic error: Only Integer literals or expressions which reduce to constant Integer are accepted as kind parameters
   --> tests/errors/continue_compilation_2.f90:287:13
    |
287 |     integer(4.2) :: ifoix
    |             ^^^ 

semantic error: Kind 6 is not supported for Complex
   --> tests/errors/continue_compilation_2.f90:289:13
    |
289 |     complex(6) :: iiocx
    |             ^ 

semantic error: Kind 3 is not supported for Integer
   --> tests/errors/continue_compilation_2.f90:291:13
    |
291 |     integer(3) :: iifix
    |             ^ 

semantic error: Kind 10 is not supported for Logical
   --> tests/errors/continue_compilation_2.f90:293:13
    |
293 |     logical(10) :: iiolx
    |             ^^ 

semantic error: Expected initialization expression for kind
   --> tests/errors/continue_compilation_2.f90:295:13
    |
295 |     complex(*) :: ksoca
    |             ^ 

semantic error: Expected initialization expression for kind
   --> tests/errors/continue_compilation_2.f90:297:13
    |
297 |     integer(*) :: ksoia
    |             ^ 

semantic error: Expected initialization expression for kind
   --> tests/errors/continue_compilation_2.f90:299:13
    |
299 |     logical(*) :: ksola
    |             ^ 

semantic error: Only Integer literals or expressions which reduce to constant Integer are accepted as kind parameters
   --> tests/errors/continue_compilation_2.f90:301:13
    |
301 |     integer('a') :: ksoix
    |             ^^^ 

semantic error: Parameter 'kvoia' is a variable, which does not reduce to a constant expression
   --> tests/errors/continue_compilation_2.f90:304:10
    |
304 |     real(kvoia) :: kvoix
    |          ^^^^^ 

semantic error: Symbol is already declared in the same scope
   --> tests/errors/continue_compilation_2.f90:306:16
    |
306 |     real(3) :: x
    |                ^ redeclaration
    |
 38 |     integer, parameter :: x = 2
    |                           ~~~~~ original declaration

semantic error: Expected initialization expression for kind
   --> tests/errors/continue_compilation_2.f90:308:10
    |
308 |     real(*) kind2_a
    |          ^ 

semantic error: Parameter attribute cannot be used with Target attribute
   --> tests/errors/continue_compilation_2.f90:310:5
    |
310 |     integer, parameter, target :: foo=4
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Unexpected args, SetExponent expects (real, int) as arguments
  --> tests/errors/continue_compilation_2.f90:23:14
   |
23 |     print *, set_exponent([1, 2, 3], 2)
   |              ^^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: shape argument specified in c_f_pointer even though fptr is not an array.
  --> tests/errors/continue_compilation_2.f90:62:38
   |
62 |     call c_f_pointer(queries_1, y_1, [2])
   |                                      ^^^ 

semantic error: shape array passed to c_f_pointer must be of rank 1 but given rank is 2
  --> tests/errors/continue_compilation_2.f90:64:38
   |
64 |     call c_f_pointer(queries_2, y_2, shape)
   |                                      ^^^^^ 

semantic error: Cannot assign to a constant variable
  --> tests/errors/continue_compilation_2.f90:66:5
   |
66 |     x = 1
   |     ^^^^^ assignment here
   |
38 |     integer, parameter :: x = 2
   |                           ~~~~~ declared as constant

semantic error: More actual than formal arguments in procedure call
  --> tests/errors/continue_compilation_2.f90:69:33
   |
69 |     call myCircle%calculateArea(circleArea, 12)
   |                                 ^^^^^^^^^^^^^^ 

semantic error: Invalid argument `end` supplied
  --> tests/errors/continue_compilation_2.f90:71:5
   |
71 |     CLOSE(end=200)
   |     ^^^^^^^^^^^^^^ 

semantic error: The first argument of `cmplx` intrinsic must be present
  --> tests/errors/continue_compilation_2.f90:73:9
   |
73 |     a = cmplx(y = 2)
   |         ^^^^^^^^^^^^ 

semantic error: Coarrays are not supported yet
  --> tests/errors/continue_compilation_2.f90:79:11
   |
79 |     val = this_image ()
   |           ^^^^^^^^^^^^^ 

semantic error: Coarrays are not supported yet
  --> tests/errors/continue_compilation_2.f90:80:5
   |
80 |     call co_sum (val, result_image=1)
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Coarrays are not supported yet
  --> tests/errors/continue_compilation_2.f90:81:9
   |
81 |     if (this_image() == 1) then
   |         ^^^^^^^^^^^^ 

semantic error: Coarrays are not supported yet
  --> tests/errors/continue_compilation_2.f90:85:5
   |
85 |     call event_query(1, 1, 1)
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Operands of comparison operator are of different types
  --> tests/errors/continue_compilation_2.f90:89:9
   |
89 |     if (i > x_2) then
   |         ^^^^^^^ 

semantic error: dim has already been specified as a positional/keyword argument to size
  --> tests/errors/continue_compilation_2.f90:93:14
   |
93 |     size_a = size(a_2, 1, dim=1)
   |              ^^^^^^^^^^^^^^^^^^^ 

semantic error: dim has already been specified as a positional/keyword argument to size
  --> tests/errors/continue_compilation_2.f90:94:14
   |
94 |     size_a = size(a_2, dim = 1, 1)
   |              ^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Only Integer literals or expressions which reduce to constant Integer are accepted as kind parameters.
  --> tests/errors/continue_compilation_2.f90:96:31
   |
96 |     size_a_2 = size(a_3, kind=kindvar, dim=1)
   |                               ^^^^^^^ 

semantic error: Only Integer literals or expressions which reduce to constant Integer are accepted as kind parameters.
  --> tests/errors/continue_compilation_2.f90:97:31
   |
97 |     size_a_2 = size(a_3, kind=kindvar)
   |                               ^^^^^^^ 

semantic error: Atomic operations are not supported yet
  --> tests/errors/continue_compilation_2.f90:99:5
   |
99 |     call atomic_add (atom[1], this_image())
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Atomic operations are not supported yet
   --> tests/errors/continue_compilation_2.f90:100:5
    |
100 |     call atomic_add (atom[2], this_image())
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Type-spec cannot contain an asterisk for a type parameter
   --> tests/errors/continue_compilation_2.f90:102:14
    |
102 |     print *, [character(*) :: "a", "b", "ball", "cat"]
    |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Different `character` lengths 1 and 4 in array constructor
   --> tests/errors/continue_compilation_2.f90:104:25
    |
104 |     print *, ["a", "b", "ball", "cat"]
    |                         ^^^^^^ 

semantic error: Different `character` lengths 2 and 5 in array constructor
   --> tests/errors/continue_compilation_2.f90:105:27
    |
105 |     print *, ["a1", "b1", "ball1", "cat1"]
    |                           ^^^^^^^ 

semantic error: Element in `real(8)` array constructor is `real(4)`
   --> tests/errors/continue_compilation_2.f90:108:20
    |
108 |     print *, [x_3, [1., 2.]]
    |                    ^^^^^^^^ 

semantic error: Element in `integer(4)` array constructor is `real(4)`
   --> tests/errors/continue_compilation_2.f90:110:18
    |
110 |     print *, [1, 2.]
    |                  ^^ 

semantic error: Element in `integer(4)` array constructor is `real(4)`
   --> tests/errors/continue_compilation_2.f90:112:18
    |
112 |     print *, [1, [1., 2.]]
    |                  ^^^^^^^^ 

semantic error: Argument of intrinsic must be an integer
   --> tests/errors/continue_compilation_2.f90:114:14
    |
114 |     print *, dfloat(y_3)
    |              ^^^^^^^^^^^ 

semantic error: Invalid argument type for `dim` or `mask`
   --> tests/errors/continue_compilation_2.f90:116:29
    |
116 |     print *, sum([1, 2, 3], 1.1)
    |                             ^^^ 

semantic error: `mask` argument to `Sum` must be a scalar or array of logical type
   --> tests/errors/continue_compilation_2.f90:118:32
    |
118 |     print *, sum([1, 2, 3], 1, 1.1)
    |                                ^^^ 

semantic error: `dim` argument to `Sum` must be a scalar and of integer type
   --> tests/errors/continue_compilation_2.f90:120:37
    |
120 |     print *, sum([1, 2, 3], .true., 1.1)
    |                                     ^^^ 

semantic error: Too many arguments to call `dint`
   --> tests/errors/continue_compilation_2.f90:122:13
    |
122 |     print*, dint(1.0_8, 8)
    |             ^^^^^^^^^^^^^^ 

semantic error: Too many arguments to call `dint`
   --> tests/errors/continue_compilation_2.f90:123:13
    |
123 |     if (abs(dint(1.0_8, 8) - 1.0_8) > 10e-5 ) error stop
    |             ^^^^^^^^^^^^^^ 

semantic error: Argument 1 of dlgama must be of double precision real type
   --> tests/errors/continue_compilation_2.f90:125:14
    |
125 |     print *, dlgama(2.7)
    |              ^^^^^^^^^^^ 

semantic error: Too many arguments to call `dnint`
   --> tests/errors/continue_compilation_2.f90:127:13
    |
127 |     print*, dnint(1.0_8, 8)
    |             ^^^^^^^^^^^^^^^ 

semantic error: Too many arguments to call `dnint`
   --> tests/errors/continue_compilation_2.f90:128:13
    |
128 |     if (abs(dnint(1.0_8, 8) - 1.0_8) > 10e-5 ) error stop
    |             ^^^^^^^^^^^^^^^ 

semantic error: Do concurrent loop variable `j` cannot be part of local expression
   --> tests/errors/continue_compilation_2.f90:130:30
    |
130 |     do concurrent (j = 1:Ny) local(i_1, j)
    |                              ^^^^^^^^^^^^^ 

semantic error: Assignment to loop variable `i_2` is not allowed
   --> tests/errors/continue_compilation_2.f90:137:9
    |
137 |         i_2 = j_1 + 1
    |         ^^^ 

semantic error: Step expression (Increment) in DO loop cannot be zero
   --> tests/errors/continue_compilation_2.f90:143:20
    |
143 |     do i_2 = 1, 5, 0
    |                    ^ 

semantic error: Arguments to dprod must be real(4)
   --> tests/errors/continue_compilation_2.f90:147:13
    |
147 |     print*, dprod(4.23_8, 4.3_8)
    |             ^^^^^^^^^^^^^^^^^^^^ 

semantic error: Unexpected args, Dreal expects (complex64) as arguments
   --> tests/errors/continue_compilation_2.f90:150:14
    |
150 |     print *, dreal(a_4)
    |              ^^^^^^^^^^ 

semantic error: Incorrect number of arguments passed to the 'complex' intrinsic. It accepts exactly 2 arguments.
   --> tests/errors/continue_compilation_2.f90:152:11
    |
152 |     a_5 = complex(1)
    |           ^^^^^^^^^^ 

semantic error: Argument of intrinsic must be an integer
   --> tests/errors/continue_compilation_2.f90:154:14
    |
154 |     print *, float(y_4)
    |              ^^^^^^^^^^ 

semantic error: Invalid argument `start` supplied
   --> tests/errors/continue_compilation_2.f90:156:5
    |
156 |     FLUSH(unit=10, start=100)
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Type mismatch in argument at argument (1); passed `real` to `integer`.
   --> tests/errors/continue_compilation_2.f90:158:16
    |
158 |     print *, f(42.9)
    |                ^^^^ 

semantic error: Argument to Ichar must have length one
   --> tests/errors/continue_compilation_2.f90:160:13
    |
160 |     print*, ichar("okay")
    |             ^^^^^^^^^^^^^ 

semantic error: first argument of `idint` must have kind equals to 8
   --> tests/errors/continue_compilation_2.f90:162:14
    |
162 |     print *, idint(4.5)
    |              ^^^^^^^^^^ 

semantic error: first argument of `ifix` must have kind equals to 4
   --> tests/errors/continue_compilation_2.f90:164:14
    |
164 |     print *, ifix(4.23_8)
    |              ^^^^^^^^^^^^ 

semantic error: Different shape for array assignment on dimension 1(1 and 3)
   --> tests/errors/continue_compilation_2.f90:167:5
    |
167 |     arr1 = [1, 2, 3]
    |     ^^^^^^^^^^^^^^^^ 

semantic error: Different shape for array assignment on dimension 3(1 and 2)
   --> tests/errors/continue_compilation_2.f90:173:5
    |
173 |     idaa2_y = idaa2_x
    |     ^^^^^^^^^^^^^^^^^ 

semantic error: Shapes for operands are not conformable
   --> tests/errors/continue_compilation_2.f90:176:14
    |
176 |     print *, idlalb1_x .neqv. [.true., .true.]
    |              ^^^^^^^^^        ^^^^^^^^^^^^^^^^ 

semantic error: Shapes for operands are not conformable
   --> tests/errors/continue_compilation_2.f90:180:14
    |
180 |     print *, idlalb2_x1 .neqv. idlalb2_x2
    |              ^^^^^^^^^^        ^^^^^^^^^^ 

semantic error: Incompatible ranks 2 and 1 in assignment
   --> tests/errors/continue_compilation_2.f90:183:5
    |
183 |     iraa1_arr1 = [1, 2, 3]
    |     ^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Incompatible ranks 1 and 3 in assignment
   --> tests/errors/continue_compilation_2.f90:187:5
    |
187 |     iraa2_arr3 = iraa2_arr1
    |     ^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: the first array argument to `where` must be of type logical
   --> tests/errors/continue_compilation_2.f90:190:11
    |
190 |     where([1, 2, 3, 4, 5]) iatw1_b = 1
    |           ^^^^^^^^^^^^^^^ 

semantic error: the first array argument to `where` must be of type logical
   --> tests/errors/continue_compilation_2.f90:197:11
    |
197 |     where(iatw2_i1) iatw2_b = 1
    |           ^^^^^^^^ 

semantic error: the argument to `where` must be an array
   --> tests/errors/continue_compilation_2.f90:202:11
    |
202 |     where(.true.) itw1_b = 12121
    |           ^^^^^^ 

semantic error: the argument to `where` must be an array
   --> tests/errors/continue_compilation_2.f90:206:11
    |
206 |     where(1) itw2_b = 12121
    |           ^ 

semantic error: the argument to `where` must be an array
   --> tests/errors/continue_compilation_2.f90:210:11
    |
210 |     where(max(1.33, 2.67)) itw3_b = 12121
    |           ^^^^^^^^^^^^^^^ 

semantic error: Unexpected args, Radix expects (int) or (real) as arguments
   --> tests/errors/continue_compilation_2.f90:217:14
    |
217 |     print *, radix((2.4, 1.0))
    |              ^^^^^^^^^^^^^^^^^ 

semantic error: datan2 was declared as a variable, it can't be called as a function
   --> tests/errors/continue_compilation_2.f90:222:14
    |
222 |     print *, datan2(x,y)
    |              ^^^^^^^^^^^ help: use the compiler option "--implicit-interface" to use intrinsic functions

semantic error: datan2 was declared as a variable, it can't be called as a function
   --> tests/errors/continue_compilation_2.f90:223:12
    |
223 |     if(abs(datan2(x,y) - 0.59941916594660438) > 1d-6) error stop
    |            ^^^^^^^^^^^ help: use the compiler option "--implicit-interface" to use intrinsic functions

semantic error: `pos` argument of `ibclr` intrinsic must be non-negative
   --> tests/errors/continue_compilation_2.f90:225:14
    |
225 |     print *, ibclr(1, -2)
    |              ^^^^^^^^^^^^ 

semantic error: Kind of all the arguments of Dshiftl must be the same
   --> tests/errors/continue_compilation_2.f90:227:14
    |
227 |     print *, dshiftl(1, 1_8, 1)
    |              ^^^^^^^^^^^^^^^^^^ 

semantic error: Kind of all the arguments of Ior must be the same
   --> tests/errors/continue_compilation_2.f90:229:14
    |
229 |     print *, ior(1, 1_8)
    |              ^^^^^^^^^^^ 

semantic error: Kind of all the arguments of Ieor must be the same
   --> tests/errors/continue_compilation_2.f90:231:14
    |
231 |     print *, ieor(1, 1_8)
    |              ^^^^^^^^^^^^ 

semantic error: Kind of all the arguments of Hypot must be the same
   --> tests/errors/continue_compilation_2.f90:233:14
    |
233 |     print *, hypot(1.0, 2.7_8)
    |              ^^^^^^^^^^^^^^^^^ 

semantic error: Kind of all the arguments of Ior must be the same
   --> tests/errors/continue_compilation_2.f90:237:14
    |
237 |     print *, ior(intr8_x, intr8_y)
    |              ^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Kind of all the arguments of Iand must be the same
   --> tests/errors/continue_compilation_2.f90:241:14
    |
241 |     print *, iand(intr9_x, intr9_y)
    |              ^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Kind of all the arguments of Ieor must be the same
   --> tests/errors/continue_compilation_2.f90:245:14
    |
245 |     print *, ieor(intr10_x, intr10_y)
    |              ^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Kind of all the arguments of Hypot must be the same
   --> tests/errors/continue_compilation_2.f90:249:14
    |
249 |     print *, hypot(intr11_x, intr11_y)
    |              ^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: All arguments to max0 must be of the same type
   --> tests/errors/continue_compilation_2.f90:251:14
    |
251 |     print *, max(12, 13.94)
    |              ^^^^^^^^^^^^^^ 

semantic error: All arguments to min0 must be of the same type
   --> tests/errors/continue_compilation_2.f90:253:14
    |
253 |     print *, min(12, 13.94)
    |              ^^^^^^^^^^^^^^ 

semantic error: Unexpected args, Scale expects (real, int) as arguments
   --> tests/errors/continue_compilation_2.f90:255:14
    |
255 |     print *, scale([1, 2, 3], 2)
    |              ^^^^^^^^^^^^^^^^^^^ 

semantic error: Unexpected args, SetExponent expects (real, int) as arguments
   --> tests/errors/continue_compilation_2.f90:257:14
    |
257 |     print *, set_exponent([1, 2, 3], 2)
    |              ^^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: Non-variable expression for `iostat`
   --> tests/errors/continue_compilation_2.f90:262:5
    |
262 |     read(ici_buffer, *, iostat=ici_ios)
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: `iostat` must be scalar
   --> tests/errors/continue_compilation_2.f90:267:5
    |
267 |     read(insv_buffer, *, iostat=insv_ios(1:1))
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 

semantic error: The SHIFT argument must be less than or equal to the of SIZE argument
   --> tests/errors/continue_compilation_2.f90:269:14
    |
269 |     print *, ishftc(10, 6, 4)
    |              ^^^^^^^^^^^^^^^^ 

semantic error: The first argument of `cmplx` intrinsic is of complex type, the second argument in this case must be absent
   --> tests/errors/continue_compilation_2.f90:272:14
    |
272 |     print *, cmplx(complex_z , 1)
    |              ^^^^^^^^^^^^^^^^^^^^ 

semantic error: kind argument of `aint` intrinsic must be a scalar
   --> tests/errors/continue_compilation_2.f90:274:36
    |
274 |     print *, aint([1.0, 2.0, 3.0], [4, 4])
    |                                    ^^^^^^ 

semantic error: Type mismatch in assignment, the types must be compatible
   --> tests/errors/continue_compilation_2.f90:277:5
    |
277 |     tm1_x = "x"
    |     ^^^^^   ^^^ type mismatch (integer and string)

semantic error: Type mismatch in binary operator, the types must be compatible
   --> tests/errors/continue_compilation_2.f90:280:13
    |
280 |     tm2_x = 5 + "x"
    |             ^   ^^^ type mismatch (integer and string)

semantic error: Type mismatch in assignment, the types must be compatible
   --> tests/errors/continue_compilation_2.f90:280:5
    |
280 |     tm2_x = 5 + "x"
    |     ^^^^^       ^^^ type mismatch (integer and string)

semantic error: Variable 'foo' is not declared
   --> tests/errors/continue_compilation_2.f90:311:13
    |
311 |     print *,foo
    |             ^^^ 'foo' is undeclared

semantic error: Cannot assign to an intent(in) variable `y`
   --> tests/errors/continue_compilation_2.f90:324:5
    |
324 |     y = 99  
    |     ^ 
